website/content/en/docs/faq.md

```diff
@@ -138,6 +138,25 @@ For more documentation on enabling IPv6 with the Amazon VPC CNI, see the [docs](
 Windows nodes do not support IPv6.
 {{% /alert %}}
 
+### Why do I see extra nodes get launched to schedule pending pods that remain empty and are later removed?
```

Comment:
If the first node is reaped as you describe, I'd expect the pod to never schedule since that cycle should continue forever. I think it comes down to if the temporary startup taint is removed quickly enough to allow the pending pod to schedule before the node is reaped due to emptiness consolidation.

Reply:
I'll add a note about this case.